### PR TITLE
Payments Hub Documentation

### PR DESCRIPTION
**Pull request by takeit**
This documentation will be available at http://payments-hub.readthedocs.io/en/latest/ if the PR will be merged.

### CODE REVIEW

File: src/PH/Bundle/CoreBundle/Resources/views/index.html.twig [hunk 1]
```diff
@@ -101,11 +101,11 @@
             $.ajax({
                 type: "POST",
                 contentType: "application/json",
-                url: '/public-api/v1/subscriptions/',
+                url: '/app_dev.php/public-api/v1/subscriptions/',
                 data: JSON.stringify(data),
                 dataType: "json"
             }).done(function( data ) {
-                window.location.replace('/public-api/v1/subscriptions/'+data.token_value+'/pay/');
+                window.location.replace('/app_dev.php/public-api/v1/subscriptions/'+data.token_value+'/pay/');
```
Comment (ahilles107) [inline src/PH/Bundle/CoreBundle/Resources/views/index.html.twig]: why you have here app_dev?

File: src/PH/Bundle/PayumBundle/Resources/config/actions.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 parameters:
     payum.template.sepa_obtain_bank_details: '@@PHPayum/Action/obtain_sepa_bank_account.html.twig'
+    payum.template.sepa_mandate_confirmation: '@@PHPayum/Action/sepa_mandate_confirmation.html.twig'
```
Comment (ahilles107) [inline src/PH/Bundle/PayumBundle/Resources/config/actions.yml:3]: `@@` is correct?

Comment (takeit) [reply src/PH/Bundle/PayumBundle/Resources/config/actions.yml:3]: yes

File: src/PH/Bundle/CoreBundle/Resources/views/index.html.twig (hunk 1 above)
Comment (takeit) [reply src/PH/Bundle/CoreBundle/Resources/views/index.html.twig]: I changed it to test, will revert.

**Pull request merged**